server.py

```diff
@@ -159,6 +177,32 @@ def _get_url_type(url:str):
             logging.error(f"url {url} is not a playlist or video!")
             return UrlType.UNKNOWN
 
+def handle_cache_play():
+    # Get all the videos in the cache
+    cache_videos =  video_cache.video_id_to_path
+
+    # Loop through each video in the cache
+    for _ , video in cache_videos.items():
+
+        # Store the current playing video information 
+        current_video_dict["title"] = video.title
+        current_video_dict["thumbnail"] = video.thumbnail
+
+        # Get the file path of the video to stream
+        file_path = video.file_path
+
+        # Start thread to stream the video and provide a response
+        try:
+            response = create_ffmpeg_stream(file_path, State.PLAYING)
+            
+            # if the video ended on its own, continue to the next video, otherwise break out of the loop
+            if response != 0:
+                break
+        
+        except Exception as e:
+            logging.exception(e)
+            raise HTTPException(status_code=500, detail="check logs")
```

Comment:
if we are just gonna catch and then re raise the exception we might as well not have the try/catch at all

Reply:
resolved